### DISCUSSION TITLE
"Proof of Contribution" in social thesis

### DISCUSSION THREAD

**Discussion by 0xsingular** [pkgxdev/pkgx#887] [Ideas]
As far as we know as a simple form the tea protocol will focus on (financial) support for developers working/contributing to open source projects.
Are there plans for the future or the idea of adding another side to the tea protocol that will allow projects to receive social, community support and community involvement in building and supporting projects?

This way developers who come up with an original concept that is needed by the web3 ecosystem in general will have full support for the realisation of this project. The social part should not include TEA rewards, already each project in an open form can set the benefits for social contributions. It would probably be achievable through a combination of Proof of Contribution consensus and [syndicate’s programmable smart wallets](https://docs.syndicate.io/features/programmable-wallets) + [modular smart contracts](https://docs.syndicate.io/features/smart-contracts). In theory such involvement would build a stronger community around the projects, and developers would be more excited to experiment with new ideas knowing that they would have a protocol (tea) through which they could connect with people who could provide the full support needed to realize these ideas.

**Comment by mxcl** [comment]
Probs this should be asked at https://github.com/teaxyz/white-paper/discussions?

**Reply by 0xsingular** [reply]
Done! https://github.com/teaxyz/white-paper/discussions/110
Sorry if I caused a mess with this topic here.
Thank you!
